pyproject.toml

```diff
@@ -11,6 +11,7 @@ python-dotenv = "^0.14.0"
 requests = "^2.25.1"
 
 [tool.poetry.dev-dependencies]
+pytest = "^6.2.2"
```

Comment:
We haven't got a dependency on selenium - we should add that here too

Reply:
Dependencys have now been added.